source/manuals/programming-languages/js.html.md.erb

```diff
@@ -24,47 +24,95 @@ owner_slack: '#frontend'
 
 ## Linting
 
-We follow [standardjs](https://standardjs.com/), an opinionated JavaScript linter. In cases where [standard conflicts with a service's browser support](#when-to-use-standardx) we use [standardx][].
+As a base, we follow conventions established by the [standardjs][] project. 
+They allow us to be consistent about how we write code while reducing the time spend debating which linting rules to pick.
 
-All JavaScript files follow its conventions, and it typically runs on CI to ensure that new pull requests are in line with them.
+Depending on their needs, projects may complement this initial set of rules with extra linting, for example:
 
-[standardx]: https://github.com/standard/standardx
+- other [ESLint][] plugins, like [es-x][] to check API compatibility with browsers your project supports
+- [Prettier][] to enforce further code formatting conventions
+
+You should be cautious to only amend the initial set of rules to resolve compatibility issues, and not as a means to adjust rules to individual preferences.
+
+**Why**: Linting ensures consistency in the codebase and picks up on well-known issues. Using an opinionated set of rules allows us to limit time spend picking rules, focusing instead on getting consistency, which is more important.
 
-### Running standard manually
+[standardjs]: https://standardjs.com/
+[ESLint]: https://eslint.org/
+[es-x]: https://github.com/eslint-community/eslint-plugin-es-x
+[Prettier]: https://prettier.io/docs/en/cli.html
 
-To check the whole codebase, run:
+### Tools
 
-```bash
+#### StandardJS's command line interface
+
+If you're not looking to use make any amends to the StandardJS conventions, you can use [StandardJS' `standard` command line interface][standard-cli] to lint files in your repository without extra set up.
+
+```
 npx standard
 ```
-### Running standard in your editor
 
-Easier than running standard manually is to install it as a plugin in your editor. This way, it will run automatically while you work, catching errors as they happen on a per-file basis.
+[standard-cli]: https://standardjs.com/#usage
 
-There are [official guides for most of the popular editors](https://standardjs.com/index.html#are-there-text-editor-plugins).
+#### standardx
 
-### When to use standardx
-
-You should use [standardx][] when the standard's rule set conflicts with the browsers your project supports. For example, the [no-var](https://eslint.org/docs/rules/no-var) rule in standard - which prefers `let` or `const` over `var` - prevents JavaScript usage in versions of Internet Explorer < 11. Adopting this rule would mean explicitly breaking JavaScript for those browsers.
+If the StandardJS's rule set conflicts with the browsers your project supports, you can use [standardx][] to amend which rules are running. 
```

Comment:
I think this should be either `the StandardJS rule set` or `StandardJS's rule set`:
```suggestion
If the StandardJS rule set conflicts with the browsers your project supports, you can use [standardx][] to amend which rules are running. 
```